frontend/js/redux/actions/result-actions.js

```diff
@@ -101,6 +109,8 @@ export function createElectionReminder(SurveyResultId, payload, callback) {
         const body = {};
         payload.email && (body.email = payload.email);
         payload.phone && (body.phone = payload.phone);
+        payload.name && (body.name = payload.name);
```

Comment:
whats the difference between this and just assigning it directly to body in declaration?